Gems/Atom/RHI/Vulkan/Code/Source/RHI/PhysicalDevice.cpp

```diff
@@ -284,6 +288,10 @@ namespace AZ
                 (m_vulkan12Features.separateDepthStencilLayouts));
             m_features.set(static_cast<size_t>(DeviceFeature::DescriptorIndexing), VK_DEVICE_EXTENSION_SUPPORTED(context, EXT_descriptor_indexing));
             m_features.set(static_cast<size_t>(DeviceFeature::BufferDeviceAddress), VK_DEVICE_EXTENSION_SUPPORTED(context, EXT_buffer_device_address));
+            // Disable memory budget extension for now since it's crashing the driver when querying heap budgets.
```

Comment:
It would be nice to add the crash related details in the comment here so if this fixed in future we will know when to remove this or just in general have better context around this issue.